c2corg_ui/static/js/map/map.js

```diff
@@ -404,14 +404,28 @@ app.MapController.prototype.createStyleFunction_ = function() {
  * @private
  */
 app.MapController.prototype.createPointStyle_ = function(feature, resolution) {
+  var module = feature.get('module');
+  var path;
+  var imgSize;
   var type = /** @type {string} */ (feature.get('module'));
   if (type === 'waypoints' && feature.get('type')) {
     type = /** @type {string} */ (feature.get('type'));
   }
   var id = /** @type {number} */ (feature.get('documentId'));
   var highlight = /** @type {boolean} */ (!!feature.get('highlight'));
   var scale = highlight ? 1 : 0.5;
-  var imgSize = highlight ? 32 : 16;
+
+  if (module === 'waypoints') {
+    imgSize = highlight ? 40 : 24;
+    path = '/documents/waypoints/' + type + '.svg';
+  } else if (module === 'images') {
```

Comment:
Do you really need to make a special case for images since it's the same path and size than the default?